python/pyspark/pandas/groupby.py

```diff
@@ -3327,15 +3326,9 @@ def normalize_keyword_aggregation(
     Examples
     --------
     >>> normalize_keyword_aggregation({'output': ('input', 'sum')})
-    (OrderedDict([('input', ['sum'])]), ['output'], [('input', 'sum')])
+    (defaultdict(<class 'list'>, {'input': ['sum']}), ['output'], [('input', 'sum')])
```

Comment:
Then we maybe should address the description in the line 3310-3311 ?

From
`Transforms from the new ``Dict[str, NamedAgg]`` style kwargs to the old OrderedDict[str, List[scalar]]].`
to
`Transforms from the new ``Dict[str, NamedAgg]`` style kwargs to the old defaultdict[str, List[scalar]]].`

?